WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerViewModel.kt

```diff
@@ -87,8 +116,38 @@ class StoryComposerViewModel @Inject constructor(
         postEditorAnalyticsSession.setOutcome(CANCEL)
     }
 
+    private fun updateStoryPostWithChanges() {
+        editPostRepository.postChanged.observe(this, Observer {
```

Comment:
Hi @malinajirka so I needed to make this `ViewModel` a `LifecycleOwner` because we need to observe changes in the `EditPostRepository` and do the relevant updates. I utilized the approach that was present in the `PostListMainViewModel` I remember you saying that a lot of approaches taken there could be improved so let me know if there's a better way to do this. Thanks 🙏

Reply:
Ohh I see. Thanks for the explanation. Even though I'm not convinced observing within VM is a good practice I can't think of an easy enough solution right now. So I'd keep it as is.